### PR TITLE
enhance secondary nav

### PR DESCRIPTION
**Pull request by cvan**
### before
image

### after
image

### CODE REVIEW

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs [hunk 1]
```diff
@@ -3,38 +3,45 @@
...
     url: 'community/',
-    title: 'Community',
+    text: 'Community',
     slug: 'community'
+  },
+  {
+    url: config.github.aframe.url,
+    text: 'GitHub',
+    slug: 'github',
+    title: 'A-Frame Slack (sign up)'
```
Comment (ngokevin) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: remove

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs [hunk 2]
```diff
@@ -3,38 +3,45 @@
...
+  },
+  {
+    url: config.slack.aframe.signup_url,
+    text: 'Slack',
+    slug: 'github',
```
Comment (ngokevin) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: remove

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs [hunk 3]
```diff
@@ -65,7 +72,7 @@ links = links.map(function (link) {
 
   <ul id="nav" class="nav">
     <% links.forEach(function (link) { %>
-      <li class="nav-item"><a href="<%= url_for(link.url) %>" class="nav-link nav-link-docs<%= link.current ? ' current' : '' %>"<%- link.external ? ' target="blank" rel="external"' : '' %>><span><%= link.title %></span></a></li>
+      <li data-nav-slug="<%= link.slug %>" class="nav-item"><a href="<%= url_for(link.url) %>" class="nav-link nav-link-docs<%= link.current ? ' current' : '' %>"<%- link.external ? ' rel="external"' : '' %><% if (link.title) { %> title="<%= link.title %>"<% } %>><span><%= link.text %></span></a></li>
```
Comment (ngokevin) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: could break this line up

File: _config.yml [hunk 4]
```diff
@@ -125,6 +125,11 @@ github:
     repo: aframe-site
     url: https://github.com/aframevr/aframe-site/
 
+slack:
+  aframe:
+    url: https://aframevr.slack.com
```
Comment (ngokevin) [inline _config.yml]: Used?

**Comment by ngokevin** [comment]
r+wc

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs (hunk 1 above)
Comment (cvan) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: oops, good call

File: themes/aframe/layout/partials/secondary/sidebar_header.ejs (hunk 2 above)
Comment (cvan) [inline themes/aframe/layout/partials/secondary/sidebar_header.ejs]: good catch

**Comment by cvan** [comment]
updated, thanks

**Pull request merged**